### ISSUE TITLE
Add read-only One Way field with Change Direction button to presets that have an implicit direction

### ISSUE THREAD

**Issue by EvanCarroll** [openstreetmap/iD#9279]
### How to reproduce the issue?

streams and drains have a direction. we currently can not reverse the direction of them.

### Screenshot(s) or anything else?

![d](https://user-images.githubusercontent.com/19922/188995467-593d868d-4664-4191-bba5-7b701eea8569.png)

**Comment by 1ec5** [comment]
You can right-click on the waterway on the map and click on Reverse, or press <kbd>V</kbd>. However, waterway presents don’t have the One Way field, which happens to have a Reverse Direction button in it, because waterways aren’t explicitly tagged with the `oneway=yes` or `oneway=no` tag.

**Comment by EvanCarroll** [comment]
Perhaps it would be a nice add, even if the tag didn't actually exist. Even if that's how it exists in the DB, from a UX perspective that's confusing. Also not sure how the user is supposed to know about the hot-key?

**Comment by 1ec5** [comment]
Yeah, it’s unfortunate that these geometry operations are confined to the context menu but not exposed somewhere more visible. The Reverse Direction button is only part of that field because it’s related to tagging a way as being one-way. Perhaps for waterways and other presets that are inherently one-way, the One Way field could be present but checked and disabled. I’m not sure if the change would need to happen here or in [id-tagging-schema](https://github.com/openstreetmap/id-tagging-schema/) or both.

> Also not sure how the user is supposed to know about the hot-key?

The items in the context menu indicate the associated hotkey. You can also open the Help panel and click Keyboard Shortcuts (or press <kbd>?</kbd>) to see a full list. There are quite a few goodies in there.

**Comment by EvanCarroll** [comment]
> Perhaps for waterways and other presets that are inherently one-way, the One Way field could be present but checked and disabled.

That sounds great, =) but it's not ideal unless it has the `Reverse Direction` button, just so we're on the same page.

**Comment by tyrasd** [comment]
I think the idea to implement this via a readonly "pseudo" field is probably not the best idea, and could be quite confusing. Because if we were to do this, what would stop us to not also add such a field for buildings to make them square, and so on… The fields should remain limited to the tags of an object.

I would have thought that the context menu should be sufficient to discover and use the reverse operation. What would be the use cases where one might want to do this from the feature sidebar instead?
